### PR TITLE
Qli ch02

### PR DESCRIPTION
**Pull request by qianliok**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,5 @@
 *.class
+*.classpath
```
Comment (qianliok) [inline .gitignore]: trying to gitignore the `.classpath` file, but somehow it shows up on my git diffs ...

File: src/qli/java/com/lambdaherding/edi/qli/ch02/Calculate.java [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+				inputButton("7"), inputButton("8"), inputButton("9"),
+				binaryActionButton("/", (x, y) -> x / y), unaryActionButton("%", x -> x / 100),
+				inputButton("4"), inputButton("5"), inputButton("6"), 
+				binaryActionButton("*", (x, y) -> x * y), unaryActionButton("1/x", x -> 1 / x),
+				inputButton("1"), inputButton("2"), inputButton("3"), 
+				binaryActionButton("-", (x, y) -> x - y), unaryActionButton("+/-", x -> -x),
+				commaButton(), inputButton("0"), inputButton("."), 
+				binaryActionButton("+", (x, y) -> x + y), unaryActionButton("\u221a", x -> Math.sqrt(x)),
+				clearAll("ClearAll")
+				).forEach( b -> pan.add(b) );
```
Comment (thatguy59) [inline src/qli/java/com/lambdaherding/edi/qli/ch02/Calculate.java:62]: Hello streams.

File: src/qli/java/com/lambdaherding/edi/qli/ch02/Calculate.java [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+				JOptionPane.showMessageDialog(null, "Not enough numbers on stack", "Stack too small", JOptionPane.ERROR_MESSAGE);
+			} else {
+				double result = binaryOp.apply(data.pop(), data.pop());
+				data.push(result);
+				deleteLastNumber();
+				deleteLastNumber();
+				addToDisplay(result + ",");
+			}
+		});
+	}
```
Comment (thatguy59) [inline src/qli/java/com/lambdaherding/edi/qli/ch02/Calculate.java:134]: You went for a similar approach to me here, I was hoping I could somehow merge unaryActionButton and binaryActionButton and deal with variable numbers of input arguments but I couldn't find a way

**Review by thatguy59 — APPROVED** [review]
Nice one!

**Pull request merged**